Source/FizzWare.NBuilder/Generators/RandomGenerator.cs

```diff
@@ -72,9 +75,11 @@ public virtual decimal Next(decimal min, decimal max)
 
             int integer = rnd.Next(iMin, iMax);
             int fraction = rnd.Next(0, 4000);
-
+#if DNCORE
+            var separator = CultureInfo.CurrentCulture.NumberFormat.CurrencyDecimalSeparator;

```

Comment:
This was my first try in porting :-) thanks for the review and the feedback. We're going to make more use of NBuilder in the near future again so I'll alos look in further improvements regarding the porting and also the shared source across .Net 4 & Core